### ISSUE TITLE
Change bot to display contributors

### ISSUE THREAD

**Issue by KendallDoesCoding** [agamjotsingh18/pollitup#28]
Use [AllContributors](https://allcontributors.org/docs/en/bot/installation) instead of https://contrib.rocks/image?repo=agamjotsingh18/pollitup

**Comment by agamjotsingh18** [comment]
You want to work on this issue?

**Comment by KendallDoesCoding** [comment]
> You want to work on this issue?

Yup ofc, but first you need to add the bot/app to this repo, then it's a one min job which I can do, then it will make PR's and you gotta merge it right in!

**Comment by KendallDoesCoding** [comment]
> > You want to work on this issue?
> 
> Yup ofc, but first you need to add the bot/app to this repo, then it's a one min job which I can do, then it will make PR's and you gotta merge it right in!

Please refer to the hyperlink under "AllContributors" and download the bot/app from there.

**Comment by agamjotsingh18** [comment]
Done

**Comment by KendallDoesCoding** [comment]
@all-contributors add @agamjotsingh18 to the doc

**Comment by KendallDoesCoding** [comment]
@agamjotsingh18 We need to do this one by one or it will cause conflicts, please merge #32 and i will do it for the next profile.

**Comment by agamjotsingh18** [comment]
Hey, can you add all the contributors?

**Comment by KendallDoesCoding** [comment]
> Hey, can you add all the contributors?

Yeah, I can only do one at a time otherwise merge conflicts occur, if you want, you can use this command and create the PR too:

@all-contributors please add {@username} to the doc

**Comment by KendallDoesCoding** [comment]
@all-contributors please add @ayush25102001 to the doc

**Comment by agamjotsingh18** [comment]
Can't we change this manual thing to automatic?
Like contributors will be added automatically to the readme file

**Comment by agamjotsingh18** [comment]
@all-contributors please add @PrajwalDhule to the doc

**Comment by KendallDoesCoding** [comment]
> Can't we change this manual thing to automatic? Like contributors will be added automatically to the readme file

That's not possible unfortunately, but allcontributors is much better.

**Comment by agamjotsingh18** [comment]
@all-contributors please add @coder-mohit to the doc

**Comment by agamjotsingh18** [comment]
@all-contributors please add @Sanket1308 to the doc

**Comment by agamjotsingh18** [comment]
@all-contributors please add @jyotivakare33 to the doc

**Comment by agamjotsingh18** [comment]
@all-contributors please add @yeshwanth235 to the doc

**Comment by agamjotsingh18** [comment]
@all-contributors please add @Shubham29012001 to the doc

**Comment by agamjotsingh18** [comment]
@all-contributors please add @rsinghcodes to the doc

**Comment by KendallDoesCoding** [comment]
> Yeah, I can only do one at a time otherwise merge conflicts occur, if you want, you can use this command and create the PR too:


I told you, it needs to be done one at a time or merge conflicts occur @agamjotsingh18 .

**Comment by agamjotsingh18** [comment]
@all-contributors please add @Sanket1308 to the doc

**Comment by agamjotsingh18** [comment]
@all-contributors please add @jyotivakare33 to the doc

**Comment by agamjotsingh18** [comment]
@all-contributors please add @ronitblenz to the doc

**Comment by agamjotsingh18** [comment]
@all-contributors please add @yeshwanth235 to the doc

**Comment by agamjotsingh18** [comment]
@all-contributors please add @Shubham29012001 to the doc

**Comment by agamjotsingh18** [comment]
@all-contributors please add @rsinghcodes to the doc
